src/core/index.ts

```diff
@@ -3,11 +3,18 @@ import config from '../config';
 import { imageController } from './image/image.controller';
 import { videoController } from './video/video.controller';
 import fastifyMultipart from 'fastify-multipart';
+import { setupSwagger } from '../docs/setup-swagger';
 
-const server = fastify({ logger: true });
+function ajvPlugin(ajv: any) {
+  ajv.addFormat('binary', { type: 'string', validate: () => true });
```

Comment:
It does add `binary` format to the input, but it still doesn't render file input in Swagger UI.
That's not the ideal way of doing that, but currently there is no better way as there is still open issue for file uploads:
https://github.com/fastify/fastify-swagger/issues/47